client/state/jetpack-remote-install/actions.js

```diff
@@ -0,0 +1,50 @@
+/** @format */
```

Comment:
What do you think about adding some tests for these actions? Should be very straightforward.

Reply:
> I not convinced of the benefit of testing the action creators. 

Tests for action creators would benefit us in verifying that an action creator would return the exact action with the right structure. So if we ever decide to change something in the action creators, tests would fail, similarly to how the code that relies on the particular action creator structure would break.

> What would we be testing exactly?

Tests for action creators are pretty simple - you test whether the action creator function returns the expected object when certain parameters are passed.

> Plus, they get a run-out in the data-layer tests, and the forthcoming reducer tests.

The data layer reducer tests are expected to be testing their own set of functionality that they're solely responsible for, not the actual action structure.

Reply:
> So if we ever decide to change something in the action creators, tests would fail

There would be no way to change the action creators _without_ these tests failing. Meaning that they catch _any_ change, not just unintended consequences of some change. That gives them very limited value in my view.

Reply:
> There would be no way to change the action creators without these tests failing. 

That really depends on how the test is written - it could be written to only allow a certain shape of an object, but it could also be more flexible and require only certain properties that are essential to the action creator functionality, ignoring everything else.

Reply:
> require only certain properties that are essential to the action creator functionality

Very true. But then the tests can get out of date, and...

Anyway, we can agree to disagree here. Appreciate the discussion, though ❤️